.github/workflows/bump_plugins.yml

```diff
@@ -0,0 +1,109 @@
+name: Update plugins
+on:
+  workflow_dispatch:
```

Comment:
This mean we need to trigger it manually, which I'd prefer for the time being, once it's stable enough and the list of plugins is finalized, we can switch to schedule